src/modules/course-list/CourseList.module.tsx

```diff
@@ -29,7 +29,7 @@ export const CourseList: FC = () => {
       <h1 className="course-list__title">Course List</h1>
       <div className="courses">
         {paginatedCourses.map((course) => {
-          return <CourseItem key={course.id} courseData={course} />;
+          return <CourseCard key={course.id} courseData={course} />;
```

Comment:
To make code easier to read, you can remove `return` and add round brackets, like `course=>({CourseCard ...})`

Reply:
It's ESLint plugin rule - to make arrow function return explicit.